src/bun.js/api/server/RequestContext.zig

```diff
@@ -1804,6 +1900,71 @@ pub fn NewRequestContext(comptime ssl_enabled: bool, comptime debug_mode: bool,
 
                     return;
                 },
+                .Render => |render_body| {
+                    if (this.server) |server| {
+                        if (@hasField(@TypeOf(server.*), "dev_server")) {
```

Comment:
I'm going to be extremely surprised if this doesn't leak memory and doesn't cause crashes.